Client/Frontend/BraveVPN/BraveVPN.swift

```diff
@@ -557,11 +290,91 @@ class BraveVPN {
 
   static func clearCredentials() {
     GRDKeychain.removeGuardianKeychainItems()
-    GRDKeychain.removeKeychanItem(forAccount: kKeychainStr_SubscriberCredential)
+    GRDKeychain.removeSubscriberCredential(withRetries: 3)
```

Comment:
When is this called? When doing this are you also trying to remove the VPN credentials?